enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -1224,6 +1224,19 @@ async function fetchRequestToken(url) {
 const config = defineEmbeddingSdkConfig({ fetchRequestToken });
 ```
 
+### Customizing loader and error components
+
+You can provide your own React components for loading and error states by specifying `loaderComponent` and `errorComponent` in the config.
+
+```tsx
+import { defineEmbeddingSdkConfig } from "@metabase/embedding-sdk-react";
+
+const config = defineEmbeddingSdkConfig({
+  loaderComponent: () => <div>Analytics is loading...</div>,
+  errorComponent: ({ message }) => <div>There was an error: {message}</div>,
+});
+```
+
```

Comment:
```suggestion
```

I'm going to be changing this README to point to the docs soon.

```

Reply:
@jeff-bruemmer Thanks! I'll avoid adding more things to `README.md` in the meantime and just add them to `docs/embedding` then 🙏🏻